scipy/stats/_morestats.py

```diff
@@ -883,6 +886,26 @@ def boxcox_llf(lmb, data):
         Data to calculate Box-Cox log-likelihood for.  If `data` is
         multi-dimensional, the log-likelihood is calculated along the first
         axis.
+    axis : int, default: 0
```

Comment:
The `_axis_nan_policy_factory` decorator assumes that the first positional argument is an array that gets reduced. That's not the case here[^1], so we can't apply it directly to the public function. Instead, we apply it to a private function - but this means we need to add the documentation manually.

[^1]: if `lmb` *were* to support array arguments, then it would not be reduced along `axis`; rather, it would work elementwise like the `p` argument of `stats.quantile`. That is a feature that can be added later.